packages/light-apps/src/IdentityHeader/IdentityHeader.tsx

```diff
@@ -222,23 +222,21 @@ export class IdentityHeader extends React.PureComponent<Props, State> {
             value={address}
             withLabel={false}
           />
+          <Margin left='medium' />
+          <NavLink to='/accounts/add'>
+            Add new account
+          </NavLink>
         </Menu.Item>
         <Menu.Item>
           <Balance address={address} fontSize='medium' />
         </Menu.Item>
-        <Dropdown
-          item
-          onChange={this.handleToggleApp}
-          options={APP_OPTIONS}
-          text={stringUpperFirst(currentLocation)}
-          value={stringUpperFirst(currentLocation)}
-        />
         <Dropdown
           icon='setting'
           item
-          text='Settings'
+          text='Manage account &nbsp;' /* TODO add margin to the icon instead */
```

Comment:
```suggestion
          text={<Margin as='span' right='small'>Manage account</Margin>}
```

I didn't test it, but should work

Reply:
prop text has type string
> TypeScript error: Type 'Element' is not assignable to type 'string'.

Reply:
ah ok, then it's SUI's fault. Let's forget about this